test/integration/composite-all-or-nothing/composite-all-or-nothing.test.ts

```diff
@@ -0,0 +1,166 @@
+import { ComponentResults, PopulationType } from '../../../src';
+import { calculate } from '../../../src/calculation/Calculator';
+import { getJSONFixture, getPopulationResultAssertion } from '../helpers/testHelpers';
+
+const MEASURE_BUNDLE: fhir4.Bundle = getJSONFixture('composite-all-or-nothing/composite-all-or-nothing-bundle.json');
+const PATIENT_COMP1_NUMER_COMP2_NUMER: fhir4.Bundle = getJSONFixture(
+  'composite-all-or-nothing/patients/patient-comp1-numer-comp2-numer-bundle.json'
+);
+const PATIENT_COMP1_NUMER_COMP2_DENOM: fhir4.Bundle = getJSONFixture(
+  'composite-all-or-nothing/patients/patient-comp1-numer-comp2-denom-bundle.json'
+);
+const PATIENT_COMP1_DENOM_COMP2_NUMER: fhir4.Bundle = getJSONFixture(
+  'composite-all-or-nothing/patients/patient-comp1-denom-comp2-numer-bundle.json'
+);
+const PATIENT_COMP1_DENOM_COMP2_DENOM: fhir4.Bundle = getJSONFixture(
+  'composite-all-or-nothing/patients/patient-comp1-denom-comp2-denom-bundle.json'
+);
+
+const COMPONENT_ONE_CANONICAL = 'http://example.com/Measure/measure-AllOrNothingComponentOne|0.0.1';
+const COMPONENT_TWO_CANONICAL = 'http://example.com/Measure/measure-AllOrNothingComponentTwo|0.0.1';
+
+describe('Composite measure all-or-nothing scoring', () => {
```

Comment:
Do we want another integration test that tests the measure report for all-or-nothing or do we feel report building is sufficiently addressed elsewhere?

Reply:
When we initially created the integration testing we decided that it would only be for `calculate` for now, not measure reports.

MeasureReport integration testing could be a problem for future us, but out of scope for now heheh